__tests__/app.test.js

```diff
@@ -45,7 +45,50 @@ describe("/api", () => {
         expect(Object.keys(response.body.api)).toEqual([
           "GET /api",
           "GET /api/topics",
+          "GET /api/articles",
         ]);
       });
   });
 });
+
+describe("/api/articles/:articles_id", () => {
+  test("GET 200: receive a the article requested by their respective id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.article.article_id).toBe(1);
+        expect(response.body.article.title).toBe(
+          "Living in the shadow of a great man"
+        );
+        expect(response.body.article.topic).toBe("mitch");
```

Comment:
.toMatchObject could be a better matcher to use here to make your code more _DRY_

Reply:
so I should check for the Id only and the for the others properties I dont need to check one by one?